spyder/widgets/variableexplorer/dataframeeditor.py

```diff
@@ -399,23 +401,103 @@ def reset(self):
         self.endResetModel()
 
 
+class FrozenTableView(QTableView):

```

Comment:
Please add a docstring to this class to explain why it's needed and where you took it from (i.e. the Qt page I referred you to)